boards/x86/panther/Kconfig.defconfig

```diff
@@ -4,6 +4,25 @@ if BOARD_PANTHER
 config BOARD
 	default "panther"
 
+if NBLE
```

Comment:
Remove this, this board does not use NBLE, NBLE is only being used for Arduino 101 with a custom Bluetooth firmware